src/API/Site/Controllers/MerchantCenter/IssuesController.php

```diff
@@ -239,10 +239,10 @@ protected function get_schema_title(): string {
 	 */
 	protected function maybe_override_issue_values( array $issue ): array {
 		$is_account_issue = MerchantStatuses::TYPE_ACCOUNT === $issue['type'];
-		if ( $is_account_issue && "Account isn't eligible for enhanced free listings" === $issue['issue'] ) {
-			$issue['issue']      = 'Show products on additional surfaces across Google through enhanced free listings';
+		if ( $is_account_issue && 'merchant_quality_low' === $issue['code'] ) {
```

Comment:
Change to compare with the issue `code`. Ref: https://developers.google.com/shopping-content/guides/account-issues#merchant_quality_low